app/Console/Commands/RussionRouletteCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Contracts\Cache\Repository as CacheRepository;
+use Illuminate\Contracts\Config\Repository as ConfigRepository;
+
+class RussionRouletteCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'game:name';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'RussionRoulette game were youser win or lose.';
```

Comment:
please provide definition for `youser` word.

Reply:
i change this